bridge_adaptivity/module/tests/test_grading_policy.py

```diff
@@ -0,0 +1,75 @@
+from ddt import data, ddt, unpack
+from django.test import TestCase
+import mock
+import pytest
+
+from module.models import GradingPolicy
+from module.policies.policy_points_earned import PointsEarnedGradingPolicy
+from module.policies.policy_trials_count import TrialsCountGradingPolicy
+
+
+GRADING_POLICY_TEST_DATA = (
```

Comment:
Need to be rewritten according to the expected parameters values.

Reply:
done, added some cases, removed unused cases